lib/types/heif.ts

```diff
@@ -11,6 +11,24 @@ const brandMap = {
   hevx: 'heic', // heic-sequence
 }
 
+function detectType(buffer: Uint8Array, start: number, end: number) {
+  const brandsDetected = new Set<keyof typeof brandMap>()
+  for (let i = start; i <= end; i += 4) {
+    const brand = toUTF8String(buffer, i, i + 4) as keyof typeof brandMap
+    if (brand in brandMap) {
+      brandsDetected.add(brand)
+    }
+  }
+
+  if (brandsDetected.has('avif')) {
+    return 'avif'
+  } else if (brandsDetected.has('heic') || brandsDetected.has('heix') || brandsDetected.has('hevc') || brandsDetected.has('hevx')) {
+    return 'heic'
+  } else if (brandsDetected.has('mif1') || brandsDetected.has('msf1')) {
+    return 'heif'
+  }
```

Comment:
I see 7 distinct checks here. Do you know if there is an easy way to convert [sample.png](https://github.com/image-size/image-size/blob/main/specs/images/valid/png/sample.png) into test files that we can use to assert these changes?